src/test/java/testresults/annual_certificates/TestPostTestResultsAnnualCertificateGenerationTrl.java

```diff
@@ -299,7 +299,11 @@ public void testResults_Annual_Certificate_Generation_Trl() {
         // Post the results, together with any alterations, and verify that they are accepted.
         testResultsSteps.postVehicleTestResultsWithAlterations(testResultRecord, alterations);
         if ("warmup test".equals(name)) {
-            //ignoring results
+            try {
```

Comment:
Can't run the code locally, so can you confirm please. Is an exception actually thrown here if statusCode is not 201?

Reply:
Added a sout to print out exception.